odetoolbox/sympy_printer.py

```diff
@@ -23,6 +23,19 @@
 from sympy.printing import StrPrinter
 
 
+def _is_zero(x):
+    r"""
+    Check if a sympy expression is equal to zero.
+
+    In the ideal case, we would like to use sympy.simplify() to do simplification of an expression before comparing it to zero. However, for expressions of moderate size (e.g. a few dozen terms involving exp() functions), it becomes unbearably slow. We therefore use this internal function, so that the simplification function can be easily switched over.
```

Comment:
```suggestion
    In the ideal case, we would like to use sympy.simplify() to
    do simplification of an expression before comparing it to
    zero. However, for expressions of moderate size (e.g. a few
    dozen terms involving exp() functions), it becomes unbearably
    slow. We therefore use this internal function, so that the
    simplification function can be easily switched over.
```